src/components/analytics/index.ts

```diff
@@ -1,20 +1,42 @@
-import ReactGA from 'react-ga4'
+import useActiveWeb3React from 'hooks/useActiveWeb3React'
+import { useEffect } from 'react'
+import { UaEventOptions } from 'react-ga4/types/ga4'
+import { RouteComponentProps } from 'react-router-dom'
 import { isMobile } from 'utils/userAgent'
+import { getCLS, getFCP, getFID, getLCP, Metric } from 'web-vitals'
+
+import GoogleAnalyticsProvider from './GoogleAnalyticsProvider'
 
 export const GOOGLE_ANALYTICS_CLIENT_ID_STORAGE_KEY = 'ga_client_id'
 const GOOGLE_ANALYTICS_ID: string | undefined = process.env.REACT_APP_GOOGLE_ANALYTICS_ID
 
 const storedClientId = window.localStorage.getItem(GOOGLE_ANALYTICS_CLIENT_ID_STORAGE_KEY)
 
+const googleAnalytics = new GoogleAnalyticsProvider()
+
+const sendEvent = (event: string | UaEventOptions, params?: any) => googleAnalytics.sendEvent(event, params)
```

Comment:
export this inline, and as a defined function (not anonymous):

```
export function sendEvent(...) {
  return googleAnalytics.sendEvent(event, params)
}
```

Same with other wrapped functions.